chandra_aca/planets.py

```diff
@@ -134,22 +133,23 @@ def get_planet_angular_sep(
     """
     from agasc import sphere_dist
 
-    if not isinstance(time, CxoTime):
-        time = CxoTime(time)
-
     if observer_position == "earth":
         eci = get_planet_eci(body, time)
         body_ra, body_dec = eci_to_radec(eci)
     elif observer_position == "chandra":
         eci = get_planet_chandra(body, time)
         body_ra, body_dec = eci_to_radec(eci)
     elif observer_position == "chandra-horizons":
-        if time.shape == ():
-            time = CxoTime([time, time + 1000 * u.s])
+        time_secs = convert_time_format(time, "secs")
+
+        if time_secs.shape == ():
+            time_secs = [time_secs, time_secs + 1000]
             is_scalar = True
         else:
             is_scalar = False
-        pos = get_planet_chandra_horizons(body, time[0], time[1], n_times=len(time))
+        pos = get_planet_chandra_horizons(
+            body, time_secs[0], time_secs[-1], n_times=len(time_secs)
```

Comment:
Note the change here from `time[1]` to `time_secs[-1]` makes the code work regardless of the number of times that were passed in.